core/src/main/resources/lib/form/section.jelly

```diff
@@ -34,13 +34,19 @@ THE SOFTWARE.
     <st:attribute name="name">
       Optional attribute to create a JSON object from this section.
     </st:attribute>
+    <st:attribute name="icon">
+      An optional URL to an icon to put before the name in the section header.
+    </st:attribute>
   </st:documentation>
 
   <st:adjunct includes="lib.form.section_" />
   <f:rowSet name="${attrs.name}">
     <j:if test="${attrs.title!=null}">
       <f:block>
         <div class="section-header">
+          <j:if test="${attrs.icon!=null}">
+            <img src="${icon}"/>
```

Comment:
What would be a way to support SVG icons through `<svg>` tags and not only as img srcs?. Also, we should provide an option to add alt text for the section header, defaulting to empty string `alt=""`.

Reply:
so the alt should be "" and not configurable imo.

this is for a quick visual display only and polluting screen readers etc with alt text is a bad design.

I thought about just adding a flag and allowing raw html in the title (so you can do whatever) but thought that would be abusable and lead to a potentialy wiered looking UI.  same for SVG, these are small icons..  YAGNI

Reply:
Sounds reasonable 👍 . You are right about the YAGNI. We can start with this and if there's a need for SVG sprite icons here we can just add a way for it later.

Reply:
Feel free to resolve this one

Reply:
I guess we should have a discussion about the hight/width attributes for scaling though?

Reply:
I would suggest setting it relative to the font size. Maybe you could try with a 1.1 or 1.2 em width and height.